resource-aggregate/pb/events.proto

```diff
@@ -113,4 +113,20 @@ message ResourceStateSnapshotTaken {
     repeated ResourceDeletePending resource_delete_pendings = 6;
     AuditContext audit_context = 100;
     EventMetadata event_metadata = 101;
+}
+
+message DeviceCloudStatusUpdated {
```

Comment:
DeviceMetadataUpdated
But do we need it? Don't we want to have it more granular - more events, e.g. DeviceStatusChanged, ...? gRPC Gateway API will provide definitely more granular events.